modules/ding_campaign/ding_campaign.module

```diff
@@ -87,51 +87,6 @@ function ding_campaign_admin_paths() {
   );
 }
 
-/**
- * Implements hook_theme().
- */
-function ding_campaign_theme($existing, $type, $theme, $path) {
```

Comment:
I don't see why this complete function should be removed. Only remove `ding_campaign_grid_wrapper` and `ding_campaign_grid`.

Reply:
You forgot changes in `ding_campaign_display()` at L114, L123 and L133-L160.